spec/clinic.v1.yaml

```diff
@@ -2188,6 +2188,20 @@ paths:
             schema:
               $ref: '#/components/schemas/GenerateMergeReport'
         description: ''
+  '/v1/clinics/{clinicId}/patients/{patientId}/connect/{providerId}':
+    parameters:
+      - $ref: '#/components/parameters/clinicId'
+      - $ref: '#/components/parameters/patientId'
+      - $ref: '#/components/parameters/providerId'
+    post:
+      summary: Connect Data Provider
+      operationId: ConnectProvider
+      responses:
+        '204':
```

Comment:
Is this consistent w/ the actual response code? Looks like its 200 OK?

Reply:
I'll need to change the spec, because the frontend expects a patient object response. Good catch.